ci/ray_ci/doc/cmd_check_api_discrepancy.py

```diff
@@ -47,6 +47,16 @@
         "head_doc_file": "doc/source/train/api/api.rst",
         "white_list_apis": {},
     },
+    "tune": {
+        "head_modules": {"ray.tune"},
+        "head_doc_file": "doc/source/tune/api/api.rst",
+        "white_list_apis": {
+            # Already documented as ray.tune.search.ConcurrencyLimiter
+            "ray.tune.search.searcher.ConcurrencyLimiter",
```

Comment:
for some reason, this particular class has two versions:
- https://github.com/ray-project/ray/blob/master/python/ray/tune/search/concurrency_limiter.py#L13
- https://github.com/ray-project/ray/blob/master/python/ray/tune/search/searcher.py#L441

we can document both but not sure why they look the same